tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -149,63 +158,100 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         return pager;
     };
 
-    fluid.tests.tooltipModuleSource = function (pager) {
-        var pageLinksTop = $("a", pager.pagerBar.locate("pageLinks"));
-        var pageLinksBottom = $("a", pager["pagerBar-1"].locate("pageLinks"));
-
-        var tooltipContents = [
-            [
-                {nodeName: "b", nodeText: "bird"},
-                {nodeName: "b", nodeText: "fish"}
-            ]
-        ];
-
+    fluid.tests.tooltipModuleSource = function (pager, sequencesOfPageSizes) {
         var sequence = [];
+        var linksList = fluid.transform(pager.dataModel, function (data) {
+            return {text: data.animal};
+        });
 
-        var assertVisibleTips = function (message, targetIds) {
+        /* Change the page size and verify whether the tooltips and links are synced accordingly */
+        fluid.transform(sequencesOfPageSizes, function (pageSize) {
```

Comment:
Instead of `fluid.transform`, use `fluid.each`. `fluid.each` is for iteration over an object or array. `fluid.transform` creates a new array or object by performing a function over each member.